jupyrdf/ipyradiant
# Support for Multiple Query Frames

## Test Procedure:
* `doit`
* Run the `RDF_to_NX_Custom.ipynb` example (can see listed in the `index.ipynb`
* Review code
* CI Green

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/zwelz3/ipyradiant/gh-59-distributed-converter-queries)

### Implements:
* [x] Capability to specify multiple queries (per type)
* [x] Build custom RDF2NX example
* [x] Add testing for custom RDF2NX process

Closes #59, Closes #67
